src/OpenRealEstate.Core/Residential/ResidentialListing.cs

```diff
@@ -1,8 +1,8 @@
-﻿using System;
+using System;

```

Comment:
nitpic

Reply:
no actual change here, i dont know what is going on

Reply:
Ok - is it needed though?

Reply:
datetime

Reply:
I suck at this Pr reviewing stuff :)